app/src/androidTest/java/ch/epfl/sdp/drone3d/ui/mission/ItineraryCreateActivityTest.kt

```diff
@@ -103,11 +106,25 @@ class ItineraryCreateActivityTest {
         Intents.intended(hasComponent(MainActivity::class.java.name))
     }
 
+    @Test
+    fun goToLoginDialogWorkWhenNotLogin() {
+        `when`(authService.hasActiveSession()).thenReturn(false)
+
+        activityRule.scenario.recreate()
+
+        onView(withText(R.string.go_to_login)).perform(click())
```

Comment:
I think that you should test if the dialog is displayed
```suggestion
        onView(withText(R.string.go_to_login)).check(matches(isDisplayed()))
        onView(withText(R.string.go_to_login)).perform(click())
```